api/database/services.py

```diff
@@ -0,0 +1,144 @@
+import os
+from flask import Flask, request, jsonify
+from flask_pymongo import PyMongo
+import json
+
+def index():
+    """Display beacon info."""
+    beacon_info = {
+        "id": ".".join(reversed(request.host.split("."))),
+        "name": "Imaging beacon",
+        "type": {'group': f'{"test"}',
+                    'artifact': f'{"beacon"}',
+                    'version': f'{"0.0.0"}'},
```

Comment:
```suggestion
        "type": {'group': "test",
                    'artifact': "beacon",
                    'version': "0.0.0"},
```
removed unused f-strings